plugins/blip/basics/less/sitechange/Change.less

```diff
@@ -6,6 +6,15 @@
   @font-size: 14px;
   @line-height: 16px;
 
+  .Change-count-text {
+    display: block;
+    position: absolute;
+    font-size: 12px;
+    left: @infusion-line-start-x - ((@infusion-line-mark-size/2) + 1);
+    top: @infusion-line-y-base - 22px;
```

Comment:
Sure. I just wasn't sure if it justified a new variable/new formula, But I guess that makes sense.